modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -35,6 +35,30 @@ definition {
 		}
 	}
 
+	@description = "LPS-176607 - Verify that there is a 'cancel' button being showed"
+	@priority = 2
+	test CanCancelOrganizationEdition {
+		task ("Given a user is on the Organization page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("When a user edits an Organization") {
+			EVPOrganization.kebabMenuOrganization(organizationName = "Organization 4");
+
+			MenuItem.click(menuItem = "Edit");
+		}
+
+		task ("Then a cancel button is present") {
```

Comment:
This task could be changed to `"Then the Organization edition is canceled and the page redirects to Organization page"`